corehq/apps/domain/decorators.py

```diff
@@ -241,6 +241,22 @@ def __inner__(request, domain, *args, **new_kwargs):
     return decorator
 
 
+def api_domain_view(view):
+    """
+    Decorate this with any domain view that should be accessed via api only
```

Comment:
This is a slightly awkward sentence... I think this should be "decorate any domain view with this...", since this function is the decorator.